spec/unit/hanami/router/recognize_spec.rb

```diff
@@ -210,6 +227,21 @@
         expect(route.params).to eq({})
       end
 
+      it 'recognizes redirect' do
```

Comment:
I've noted that there seems to be no love for the `expect(route).to be_redirect` notation? Any particular reasons you folks don't like it?